tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Debugger/ProbesTests.cs

```diff
@@ -187,10 +188,11 @@ public async Task MethodProbeTest_UDS()
     }
 #endif
 
-    public void Dispose()
+    public override void Dispose()
```

Comment:
I think you can safely move this dispose method from `ProbeTests` to `TestHelpers`, instead of overriding.

Reply:
Done in c5cf6dc713be9d83186bc89361d8c9524215252e